src/event.js

```diff
@@ -610,8 +610,7 @@ jQuery.event = {
 			event,
 			{
 				type: type,
-				isSimulated: true,
-				originalEvent: {}
```

Comment:
> 1. We wouldn't need to change docs

What docs? This is for a simulated event, I don't believe there's anything special in the docs about this.

> 2. We should be back-compat if possible

There's no possible use case to be back compatible with. Did someone provide an actual scenario that I missed?

> 3. We shouldn't add needless logic & bytes

You're concerned about the delete over setting to an empty object in terms of logic and bytes?

> 4. User could explore new properties of the original event when they comes up, without for us to do anything or change our stub

That's impossible, considering it's been an empty object forever.

> 5. There is already possible use-cases, users don't use them because they can't which doens't mean they aren't useful - http://jsfiddle.net/rs5jn1bq/

And are the properties legitimate considering they're not even from the correct event type?

> 6. Consistency, this should be important and it correlates with basic concepts of the DOM events described in the spec

Consistency with what?